tests/functional_tests/add_a_new_provider/test_chambers_contact_details.py

```diff
@@ -0,0 +1,207 @@
+    page.get_by_role("textbox", name="DX centre").fill("Chambers Centre")
+    page.get_by_role("button", name="Submit").click()
+
+    # Should redirect to create provider page or success page
+    # Check that we're no longer on the contact details page
+    expect(page.get_by_role("heading", name="Add chambers contact details")).not_to_be_visible()
+
+    # Should see the success flow (depending on what main.create_provider does)
+    current_url = page.url
+    assert "add-contact-details" not in current_url
+
+
+@pytest.mark.usefixtures("live_server")
+def test_chambers_successful_form_submission_all_fields(page: Page):
+    """Test successful chambers form submission with all fields filled."""
+    navigate_to_chambers_contact_details(page)
+
+    # Fill all fields
+    page.get_by_role("textbox", name="Address line 1").fill("123 Chambers Street")
+    page.get_by_role("textbox", name="Address line 2 (optional)").fill("Lincoln's Inn")
+    page.get_by_role("textbox", name="Address line 3 (optional)").fill("Legal Quarter")
+    page.get_by_role("textbox", name="Address line 4 (optional)").fill("Barristers' Row")
+    page.get_by_role("textbox", name="Town or city").fill("Chambers City")
+    page.get_by_role("textbox", name="County (optional)").fill("Chambers County")
+    page.get_by_role("textbox", name="Postcode").fill("CH1 2MB")
+    page.get_by_role("textbox", name="Telephone number").fill("01234567890")
+    page.get_by_role("textbox", name="Email address").fill("contact@testchambers.com")
+    page.get_by_role("textbox", name="DX number").fill("DX789123")
+    page.get_by_role("textbox", name="DX centre").fill("Chambers Centre")
+    page.get_by_role("button", name="Submit").click()
+
+    # Should redirect to create provider page or success page
+    # Check that we're no longer on the contact details page
+    expect(page.get_by_role("heading", name="Add chambers contact details")).not_to_be_visible()
+
+    # Should see the success flow
+    current_url = page.url
+    assert "add-contact-details" not in current_url
+
+
+@pytest.mark.usefixtures("live_server")
+def test_chambers_optional_fields_not_required(page: Page):
+    """Test that optional fields don't prevent chambers form submission."""
+    navigate_to_chambers_contact_details(page)
+
+    # Fill required fields, leave optional fields empty
+    page.get_by_role("textbox", name="Address line 1").fill("123 Chambers Street")
+    page.get_by_role("textbox", name="Town or city").fill("Chambers City")
+    page.get_by_role("textbox", name="Postcode").fill("CH1 2MB")
+    page.get_by_role("textbox", name="Telephone number").fill("01234567890")
+    page.get_by_role("textbox", name="Email address").fill("contact@testchambers.com")
+    page.get_by_role("textbox", name="DX number").fill("DX789123")
+    page.get_by_role("textbox", name="DX centre").fill("Chambers Centre")
+    # Leave optional fields empty: address_line_2-4, county
```

Comment:
telephone number, email and both DX fields are also optional fields, can this test be updated to reflect that